### ISSUE TITLE
ansible_local - does not work with ansible 2.0.1.0 installed on guest

### ISSUE THREAD

**Issue by COLABORATI** [mitchellh/vagrant#7166]
```
==> default: Running provisioner: ansible_local...
The Ansible software could not be found! Please verify that Ansible is correctly installed on your guest system.
```

But ansible is installed on the guest: 

```
vagrant@monitor:~$ ansible --version
ansible 2.0.1.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = Default w/o overrides
```

I just want to add that the root of this problem is that vagrant_local seems to have a hardcoded ansible version < 2 and this makes it impossible to run ansible roles that need version 2. 

Yes, it hurts very much.

**Comment by gildegoma** [comment]
@COLABORATI I'm very sorry that you're having a bad experience with this first release of the ansible_local provisioner. It is true, that it unfortunately suffers from very annoying bugs, but we'll try our best to keep improving it in the next Vagrant versions.

The problem you're reporting here has been actually tracked with #6793, and is originally caused by ansible/ansible#12004 (a regression introduced after that Vagrant 1.8 was released). This has been fixed in master (patch: 9dbdb9397a92d4fc489e9afcb022621df7f60d11) and will be part of the future Vagrant 1.8.2.

Closing as duplicate of #6793.

**Comment by COLABORATI** [comment]
Thanks for your attention and for your efforts!
